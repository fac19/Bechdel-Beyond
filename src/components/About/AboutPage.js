import { Link as RouterLink } from 'react-router-dom';
import React from 'react';
import Link from '@material-ui/core/Link';
import { makeStyles } from '@material-ui/core/styles';
import Typography from '@material-ui/core/Typography';

import Button from '@material-ui/core/Button';

const useStyles = makeStyles((theme) => ({
	title: {
		marginTop: '2rem',
		textAlign: 'center',
		fontSize: '2rem',
	},
	subtitle: {
		marginTop: '1rem',
		fontSize: '1rem',
		textAlign: 'center',
	},
	buttonbox: {
		display: 'flex',
		flexDirection: 'column',
		width: '20%',
		margin: '1rem auto',
	},
	buttons: {
		marginTop: '1rem',
	},
	p: {
		fontSize: '50px',
	},
}));

export default function AboutPage() {
	const classes = useStyles();

	return (
		<>
			<Typography className={classes.title} variant="h2" component="h2">
				Bechdel & Beyond
			</Typography>
			<Typography className={classes.subtitle} variant="h5" component="h5">
				A new way to discover, review & share movies that pass the Bechdel Test
				and go beyond...
			</Typography>
			<div className={classes.buttonbox}>
				<Button className={classes.buttons} variant="outlined">
					<Link component={RouterLink} to="/bechdel">
						How it works
					</Link>
				</Button>
				<Button className={classes.buttons} variant="outlined">
					<Link component={RouterLink} to="/">
						Skip to the movies
					</Link>
				</Button>
			</div>
		</>
	);
}
