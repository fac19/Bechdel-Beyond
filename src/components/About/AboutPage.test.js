import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

import AboutPage from './AboutPage';

it('should render About page correctly', () => {
	const about = renderer
		.create(
			<Router>
				<AboutPage />
			</Router>,
		)
		.toJSON();
	expect(about).toMatchSnapshot();
});
