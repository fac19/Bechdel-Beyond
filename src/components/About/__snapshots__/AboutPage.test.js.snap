// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render About page correctly 1`] = `
Array [
  <svg
    className="makeStyles-logo-1"
    fill="none"
    height="53"
    id="favicon"
    viewBox="0 0 53 53"
    width="53"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M46.7084 41.7915V23.354H41.7917V41.7915C41.7917 47.9373 46.7084 52.854 52.8542 52.854V47.9373C49.4126 47.9373 46.7084  45.2332 46.7084 41.7915Z"
      fill="#253278"
    />
    <path
      d="M23.3542 46.7083C36.2523 46.7083 46.7083 36.2523 46.7083 23.3542C46.7083 10.456 36.2523  0 23.3542 0C10.456 0 0 10.456 0 23.3542C0 36.2523 10.456 46.7083 23.3542 46.7083Z"
      fill="#90A4AE"
    />
    <path
      d="M23.3543 25.8122C24.712 25.8122 25.8127 24.7115 25.8127 23.3538C25.8127  21.9961 24.712 20.8955 23.3543 20.8955C21.9966 20.8955 20.896 21.9961 20.896 23.3538C20.896 24.7115 21.9966 25.8122 23.3543 25.8122Z"
      fill="#37474F"
    />
    <path
      d="M23.3543 17.2082C26.7486 17.2082 29.5002 14.4566 29.5002 11.0623C29.5002 7.66809 26.7486 4.9165 23.3543 4.9165C19.9601 4.9165 17.2085 7.66809 17.2085 11.0623C17.2085 14.4566 19.9601 17.2082 23.3543 17.2082Z"
      fill="#253278"
    />
    <path
      d="M23.3543 41.7917C26.7486 41.7917 29.5002 39.0401 29.5002 35.6458C29.5002 32.2516 26.7486 29.5 23.3543 29.5C19.9601 29.5 17.2085 32.2516 17.2085 35.6458C17.2085 39.0401 19.9601 41.7917 23.3543 41.7917Z"
      fill="#253278"
    />
    <path
      d="M35.6458 29.4997C39.0401 29.4997 41.7917 26.7481 41.7917 23.3538C41.7917 19.9596 39.0401 17.208 35.6458 17.208C32.2516 17.208 29.5 19.9596 29.5 23.3538C29.5 26.7481 32.2516 29.4997 35.6458 29.4997Z"
      fill="#253278"
    />
    <path
      d="M11.0626 29.4997C14.4568 29.4997 17.2084 26.7481 17.2084 23.3538C17.2084 19.9596 14.4568 17.208 11.0626 17.208C7.66833 17.208 4.91675 19.9596 4.91675 23.3538C4.91675 26.7481 7.66833 29.4997 11.0626 29.4997Z"
      fill="#253278"
    />
  </svg>,
  <h2
    className="MuiTypography-root makeStyles-title-2 MuiTypography-h2"
  >
    Bechdel & Beyond
  </h2>,
  <h5
    className="MuiTypography-root makeStyles-subtitle-3 MuiTypography-h5"
  >
    A new way to discover, review & share movies that pass the Bechdel Test and go beyond...
  </h5>,
  <hr
    className="MuiDivider-root makeStyles-line-4"
  />,
  <div
    className="makeStyles-buttonbox-5"
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-bechdelButton-6"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        <a
          className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
          href="/bechdel"
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
        >
          How it works
        </a>
      </span>
    </button>
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-homeButton-7"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        <a
          className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
          href="/"
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
        >
          Skip to the movies
        </a>
      </span>
    </button>
  </div>,
]
`;
